composer/src/main/kotlin/com/gojuno/composer/TestRun.kt

```diff
@@ -38,6 +42,8 @@ data class AdbDeviceTest(
     }
 }
 
+fun AdbDevice.sanitizedId() = id.replace(":","-")
```

Comment:
That's not good, we shouldn't duplicate implementation, it'll be easy to de-sync them